src/ccache.cpp

```diff
@@ -1513,16 +1513,16 @@ hash_nvcc_host_compiler(const Context& ctx,
 #else
     const char* compilers[] = {"gcc", "g++"};
 #endif
-    for (size_t i = 0; i < ARRAY_SIZE(compilers); i++) {
+    for (auto& compiler : compilers) {
```

Comment:
`auto&` -> `auto`

Reply:
Since I had to think about the type, I just made it explicit char* instead of auto